extensions/mssql/src/objectManagement/ui/serverPropertiesDialog.ts

```diff
@@ -0,0 +1,87 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import * as azdata from 'azdata';
+import { ObjectManagementDialogBase, ObjectManagementDialogOptions } from './objectManagementDialogBase';
+import { IObjectManagementService } from 'mssql';
+import * as localizedConstants from '../localizedConstants';
+import { ViewServerPropertiesDocUrl } from '../constants';
+import { ServerPropertiesInfo, ServerPropertiesViewInfo } from '../interfaces';
+
+export class ServerPropertiesDialog extends ObjectManagementDialogBase<ServerPropertiesInfo, ServerPropertiesViewInfo> {
+	private generalTab: azdata.window.DialogTab;
+	private generalSection: azdata.GroupContainer;
+	private nameInput: azdata.InputBoxComponent;
+	private languageDropdown: azdata.DropDownComponent;
+	private memoryInput: azdata.InputBoxComponent;
+	private operatingSystemInput: azdata.InputBoxComponent;
+
+	private memoryTab: azdata.window.DialogTab;
+	private memorySection: azdata.GroupContainer;
+	private minServerMemoryInput: azdata.InputBoxComponent;
+	private maxServerMemoryInput: azdata.InputBoxComponent;
+
+	private numberInputType: azdata.InputBoxInputType = 'number';
+
+	constructor(objectManagementService: IObjectManagementService, options: ObjectManagementDialogOptions) {
+		super(objectManagementService, options);
+		this.generalTab = azdata.window.createTab(localizedConstants.GeneralSectionHeader);
+		this.memoryTab = azdata.window.createTab(localizedConstants.MemoryText);
+	}
+
+	protected override get helpUrl(): string {
+		return ViewServerPropertiesDocUrl;
+	}
+
+	protected async initializeUI(): Promise<void> {
+		this.initializeGeneralSection();
+		this.initializeMemorySection();
+		this.dialogObject.content = [this.generalTab, this.memoryTab];
```

Comment:
I've chatted with Sai today, he is now trying to use the tabbedPanel component, l'et's wait for him.